internal/consensus/mempool_test.go

```diff
@@ -136,8 +137,10 @@ func checkTxsRange(ctx context.Context, t *testing.T, cs *State, start, end int)
 	for i := start; i < end; i++ {
 		txBytes := make([]byte, 8)
 		binary.BigEndian.PutUint64(txBytes, uint64(i))
-		err := assertMempool(t, cs.txNotifier).CheckTx(ctx, txBytes, nil, mempool.TxInfo{})
+		var rCode uint32
```

Comment:
this is always 0? maybe best to omit

Reply:
It is filled by the functor I added to the `CheckTx` call in line 141.
This was a problem in this test: `CheckTx` was returning "no error", but the `Code` field in the `CheckTx` response was being ignored by the test... so `CheckTx` was failing but the TC was thinking it was passing.
So I was chasing a problem, and it was really cumbersome before adding line 143.

Reply:
on line 141 you reference `r.Code` not `rCode`, so I still think this isn’t being used, though perhaps not intentionally.

Reply:
I'm setting `rCode` (captured by closure) to the value of `r.Code`